doc/ref/api/hvplot_api/hvplot.hvPlot.bar.ipynb

```diff
@@ -19,9 +18,72 @@
     ".. backend-styling-options:: bar\n",
     "```\n",
     "\n",
-    "## Examples\n",
+    "## Examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Simple vertical bar plot"
```

Comment:
If possible, it'd be nice if the first example could be independent of hvsampledata (e.g. with dummy data generated with pandas/numpy), so users who don't have it installed have one chance to run this one easily.